src/uiwidgets/ArrangeableContents.as

```diff
@@ -53,13 +52,9 @@ public class ArrangeableContents extends ScrollFrameContents implements DropTarg
 		addEventListener(DragEvent.DRAG_OUT, dragAndDropHandler);
 	}
 
-	public function setItemPadding(padding:uint):void {
-		itemPadding = padding;
-		arrangeItems();
-	}
-
 	override public function updateSize():void {
-		super.updateSize();
+		//super.updateSize();
+		//setWidthHeight(this.w, this.h);
```

Comment:
updateSize in the parent class didn't do what I thought it did. this area is a WIP.